app/screens/channel/channel.android.js

```diff
@@ -72,8 +74,12 @@ export default class ChannelAndroid extends ChannelBase {
     }
 }
 
-const style = StyleSheet.create({
+const getStyle = makeStyleSheetFromTheme((theme) => ({
     flex: {
         flex: 1,
     },
-});
\ No newline at end of file
+    backdrop: {
+        flex: 1,
+        backgroundColor: theme.centerChannelBg,
```

Comment:
Does `backgroundColor` introduce redraws in this case? If not, yay, but if so, is there a way of setting `componentBackgroundColor` instead (still using `theme`)?